.github/workflows/multichain-e2e.yml

```diff
@@ -0,0 +1,64 @@
+name: Multichain E2E Tests
+
+on:
+  # for now, only run on-demand
+  workflow_dispatch:
+  pull_request:
```

Comment:
remove this line before merging

Reply:
do you want it to run on every PR? consider running it the same times as the docker build tests (merges to master or with `force:integration` enabled)

Reply:
I've updated to the conditions for `docker.yml`.. unclear to me if `force:integration` will pick this up but agree with the suggested strategy